.drone.yml

```diff
@@ -425,6 +425,8 @@ steps:
     CGO_ENABLED: "1"
   image: golang:1.20.1
   name: lint-backend
+- commands:
+    - make proto-lint
```

Comment:
what is this for?

Reply:
if we need to update drone.yml the change needs to be implemented in the .star files that we use to generate drone.yml

Reply:
Sorry, completely unrelated commit.